app/Models/Backend/Category.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Models\Backend;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+
+    protected $table = "product_category";
+    protected $fillable = ['id', 'cate_name', 'cate_description', 'cate_status', 'cate_image', 'parent_id'];
+
+    /**
+     * Function list all categories
+     *
+     * @param int    $cates  cates
+     * @param int    $parent parent
+     * @param string $str    str
+     * @param int    $select select
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function cateParent($cates, $parent = 0, $str = "--", $select = 0)
```

Comment:
[rank S] what's it? Why do you put html in a model class? In model, we must avoid to include any html code.